src/System.Runtime.Numerics/tests/BigInteger/parse.cs

```diff
@@ -694,6 +720,25 @@ private static NumberFormatInfo MarkUp(NumberFormatInfo nfi)
             return nfi;
         }
 
+        private static bool FailureNotExpectedForTrailingWhite(NumberStyles ns, bool spaceOnlyTrail)
```

Comment:
This looks okay to me, but could you please add a short comment above the method explaining what/why we are doing this? I have a feeling that I am going to come back to this and forget the context, since it is a bit confusing at first glance.